### PR TITLE
Issue 264 and 266 keycloak url issues

### PR DESCRIPTION
**Pull request by maleck13**
The following should fix issues
https://github.com/RedHatWorkshops/dayinthelife-integration/issues/264
https://github.com/RedHatWorkshops/dayinthelife-integration/issues/266

**Note**
Not currently tested. Not sure how to test issue 266
Will uninstall and reinstall to test issue 264

### CODE REVIEW

File: support/install/ansible/playbooks/openshift/roles/provision_microcks/tasks/main.yml [hunk 1]
```diff
@@ -65,3 +65,8 @@
   when: configure_only == 'false'
   register: create_microcks_app
   failed_when: create_microcks_app.stderr != '' and 'already exists' not in create_microcks_app.stderr
+
+- name: patch microcks env
+  shell: "oc set env dc/microcks KEYCLOAK_URL=https://sso-{{sso_project}}.{{ ocp_apps_domain }} -n {{microcks_project}}"
```
Comment (VinayBhalerao) [inline support/install/ansible/playbooks/openshift/roles/provision_microcks/tasks/main.yml]: The KEYCLOAK_URL should be `KEYCLOAK_URL=https://sso-{{sso_project}}.{{ ocp_apps_domain }}/auth`

Comment (VinayBhalerao) [inline support/install/ansible/playbooks/openshift/roles/provision_microcks/tasks/main.yml]: The KEYCLOAK_URL should be `KEYCLOAK_URL=https://sso-{{sso_project}}.{{ ocp_apps_domain }}/auth`

**Comment by VinayBhalerao** [comment]
I cherry-picked the commits and ran against a new env..Looks good to me. I think it's ok to merge into V2.

**Comment by maleck13** [comment]
👍 I cant merge. But I tested the microcks fix on my own cluster and it has resolved the issue being seen. I am not really familiar with the other issue so may be better tested by someone who understands the usecase better

**Pull request merged**